File: programming files/datetime_convert.java
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class datetime_convert {
    public static void main(String[] args) {
        String dateStr = "2022-03-17 10:45:30";

        //Formats the string into year, month, date, hour, minute, second
        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");

        //sets the dateObj to the current date and time
        LocalDateTime dateObj = LocalDateTime.parse(dateStr, formatter);

        //formats the date into the proper format
        String formattedDate = dateObj.format(DateTimeFormatter.ofPattern("mm/dd/yyyy HH:mm:ss"));

        System.out.println(formattedDate);
    }
}

File: programming files/datetime_convert.py
from datetime import datetime

#Sets the date to 03-17-2022
date_str = "2022-03-17 10:45:30"

#Sets the object to include the hour, minute, and second.
date_obj = datetime.strptime(date_str, '%Y-%m-%d %H:%M:%S')

#Formats the date
formatted_date = date_obj.strftime('%m/%d/%Y %H:%M:%S')

print(formatted_date)
